### PR TITLE
Remove result-api leftovers

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -46,9 +46,6 @@
 
 _LOGGER = logging.getLogger("thoth.graph_refresh_job")
 
-_SOLVER_OUTPUT = os.getenv(
-    "THOTH_SOLVER_OUTPUT", "http://result-api/api/v1/solver-result"
-)
 _PACKAGE_ANALYZER_OUTPUT = os.getenv(
     "THOTH_PACKAGE_ANALYZER_OUTPUT", "http://result-api/api/v1/package-analysis-result"
```
Comment (harshad16) [inline app.py:50]: what about this @fridex ?

Comment (fridex) [reply app.py:50]: I'll drop package analyzer in a separate PR. We should substitute it with a workflow obtaining different aspects of packages.

Comment (harshad16) [reply app.py:50]: awesome, thanks

**Review by harshad16 — APPROVED** [review]
LGTM , thanks

**Pull request merged**